app/src/main/java/br/org/cesar/madmovies/movies/data/model/RemoteMovie.kt

```diff
@@ -17,4 +18,8 @@ data class RemoteMovie(
  val title: String = "",
  val vote_average: Double = 0.0,
  val vote_count: Int = 0
-)
+) {
+ fun asMovie(): Movie {
```

Comment:
It would be better in my view to have this conversion function *outside* the data class, as this is a method not fundamental to the data class itself.

In other words, an extension function. (Could be in the same file, in my view.)

Also, consider `=` syntax instead of return.

Reply:
fixed